packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -88,8 +88,15 @@ const streamMessagesApi = orchestratorApi.injectEndpoints({
                 // Send a ping message every to the websocket server to keep the connection alive
                 const pingInterval = setInterval(() => {
                     webSocket.send('__ping__');
```

Comment:
A very nice to have: also send a ping and debounce when the app loads. This way we have faster feedback that the WS is connected

Reply:
Done.